### ISSUE TITLE
Error: No valid option generated in #select!

### ISSUE THREAD

**Issue by jurijsk** [microsoft/guidance#35]
Message continues: Please post a GitHub issue since this should not happen :)

So here I am.

Raised from:
```
  File "[..]\guidance\_program_executor.py", line 94, in run
    await self.visit(self.parse_tree)
  File "[..]\guidance\_program_executor.py", line 423, in visit
    visited_children.append(await self.visit(child, inner_next_node, inner_next_next_node, inner_prev_node, node, parent_node))
  File "[..]\guidance\_program_executor.py", line 423, in visit
    visited_children.append(await self.visit(child, inner_next_node, inner_next_next_node, inner_prev_node, node, parent_node))
  File "[..]\guidance\_program_executor.py", line 390, in visit
    command_output = await command_function(*positional_args, **named_args)
  File "[..]\guidance\library\_select.py", line 138
```

Running the following from `tutorial.ipynb`:
```
import guidance
guidance.llm = guidance.llms.OpenAI("text-davinci-003")

prompt = guidance('''Is the following sentence offensive? Please answer with a single word, either "Yes", "No", or "Maybe".
Sentence: {{example}}
Answer:{{#select "answer" logprobs='logprobs'}} Yes{{or}} No{{or}} Maybe{{/select}}''')
prompt = prompt(example="I hate pineapple on pizza, so that I am ready to kill because of it") //not my actual opinion, was trying to get it to reply "Yes".
print(prompt)
```
`option_logprobs` is:
```
{' Yes': -1000, ' No': -1000, ' Maybe': -1000}
```

**Comment by slundberg** [comment]
Can you share the version you are using? I fix went out recently for this, and I can't reproduce it right now: 
<img width="945" alt="image" src="https://github.com/microsoft/guidance/assets/3740613/cb6ae2b6-ebfb-4c49-8585-1b27d10c5a0b">

**Comment by amanape** [comment]
https://github.com/microsoft/guidance/issues/36#issuecomment-1553723123

**Comment by jurijsk** [comment]
@slundberg installed today. 

Version: 0.0.47

**Comment by amanape** [comment]
This seems to have worked for me https://github.com/microsoft/guidance/issues/40#issuecomment-1554084184

**Comment by slundberg** [comment]
Thanks for reporting this! This should now be fixed in 0.0.49 :)
